### PR TITLE
Release 0.1.187

### PR DESCRIPTION
**Pull request by pvasant**

### CODE REVIEW

**Comment by pvasant** [comment]
@vkareh  PTAL

File: CHANGES.adoc [hunk 1]
```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+== 0.0.133 Jun 16 2021
+
+-  Added capabilities support to Organization
```
Comment (vkareh) [inline CHANGES.adoc:8]:
```
== 0.1.187 Jun 16 2021

- Update model to v0.0.133
** Added capabilities support to Organization
```

Comment (pvasant) [reply CHANGES.adoc]: done.

File: CHANGES.adoc [hunk 2]
```diff
@@ -3,6 +3,11 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+== 0.0.133 Jun 16 2021
```
Comment (vkareh) [inline CHANGES.adoc:6]: It should be `0.1.187`

File: CHANGES.adoc [hunk 3]
```diff
@@ -3,6 +3,11 @@
...
+
+- Update model to v0.0.132
```
Comment (vkareh) [inline CHANGES.adoc:8]: This should be `v0.0.133`

**Comment by vkareh** [comment]
LGTM

**Comment by pvasant** [comment]
@vkareh  can you please merge this. i don't have access yet in this repo

**Pull request merged**